back/engines/free/seo/app/controllers/seo/application_controller.rb

```diff
@@ -12,19 +12,19 @@ class ApplicationController < ActionController::Base
 
     def sitemap
       statuses = %w[published archived]
-      @projects = Project.select(
+      @projects = ProjectPolicy::Scope.new(nil, Project.all).resolve.select(
         :'projects.id', :'projects.slug', :'projects.visible_to',
         :'projects.updated_at', :'admin_publications.publication_status',
         :'admin_publications.publication_type', :'admin_publications.publication_id'
       ).includes(:admin_publication).where(visible_to: 'public', admin_publications: { publication_status: statuses })
-      @folders = ProjectFolders::Folder.select(
+      @folders = ProjectFolders::FolderPolicy::Scope.new(nil, ProjectFolders::Folder.all).resolve.select(
         :'project_folders_folders.id', :'project_folders_folders.slug',
         :'project_folders_folders.updated_at', :'admin_publications.publication_status',
         :'admin_publications.publication_type', :'admin_publications.publication_id'
       ).includes(:admin_publication).where(admin_publications: { publication_status: %w[published archived] })
       @initiatives = Initiative.select(:slug, :updated_at, :publication_status).where(publication_status: statuses)
       @pages = StaticPage.select(:slug, :updated_at)
-      @ideas = Idea.select(:slug, :updated_at, :project_id).where(project_id: @projects.map(&:id))
+      @ideas = IdeaPolicy::Scope.new(nil, Idea.all).resolve.select(:slug, :updated_at, :project_id).where(project_id: @projects.map(&:id))
```

Comment:
Seems like we were exposing (the existence of) all kinds of private stuff through the Sitemap, which was not using the policies at all.